lib/registrant.rb

```diff
@@ -0,0 +1,39 @@
+class Registrant
+    attr_reader :name, 
+                :age, 
+                :permit,
+                :license_data
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @permit = permit
+        @license_data = { 
+            written: false, 
+            license: false, 
+            renewed: false
+        }
+    end
+
+    def permit?
+        @permit
+    end
+
+    def earn_permit
+        @permit = true
+        return
+    end
+
+    def has_permit?
```

Comment:
We already have this method, it's `.permit?` both of these methods are currently doing the same thing, only need one of them.